src/main/scala/com/modelfabric/sparql/stream/client/Builder.scala

```diff
@@ -138,6 +139,7 @@ object Builder {
       import GraphDSL.Implicits._
       import endpoint._
 
+      // JC: why all stages in sparqlQueryFlow are async, but not here? do we need to make all of them async?
```

Comment:
No it is not necessary, but it will work either way. As I have said, we could tweak this while doing performance tuning.